QueueImplementation/Business.UnitTest/Queue/LinkedListQueueTest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Business.Queue;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Business.UnitTest.Helper
+{
+	/// <summary>
+	/// Represents tests for <see cref="LinkedListQueue<T>"/>.
+	/// </summary>
+	/// <owner>Anton Petrenko</owner>
+	[TestClass]
+	public sealed class LinkedListQueueTest : QueueTestBase
```

Comment:
Why do the Array implementations contain more tests than Linked-List?

Reply:
Because the logic of adding new element to the array is a bit complex than to the Linked list. Therefore, it was necessary to add more tests.

Reply:
But you have the same public interface and number of tests should be the same, isn't it?

Reply:
The difference in tests is caused by code coverage result.
In order to have better coverage result I added these additional tests.

Reply:
It's not possible to move this additional logic of tests to the some base Queue test class. because of differences in realization. 
So, I left tests in ArrayQueueTest and ArrayStackTest as they are.